proficere/accounts/templates/accounts/navbar.html

```diff
@@ -1,37 +1,41 @@
 {% load static %}
+{% load extraAuth %}
```

Comment:
I think we should rename "extraAuth" to "groupAuth"

Reply:
Quit Nit Picking we have deadlines.